modules/rostests/apitests/shell32/ItemIDList.cpp

```diff
@@ -11,11 +11,45 @@
 
 enum { DIRBIT = 1, FILEBIT = 2 };
 
+static BYTE GetPIDLType(LPCITEMIDLIST pidl)
+{
+    return pidl && pidl->mkid.cb >= 3 ? (pidl->mkid.abID[0] & 0x7F) : 0;
+}
+
+struct FS95
```

Comment:
A doxygen comment block to document what this is about would be welcome.

Reply:
`GetPIDLType`...returns the type of pidl. `PIDLTYPE type;` in pidl.h but several of the PT_ values there are incorrect and have horrible names.